aliceO2/NamespaceNamingCheck.cpp

```diff
@@ -11,7 +11,6 @@
 #include "clang/AST/ASTContext.h"
 #include "clang/ASTMatchers/ASTMatchFinder.h"
 #include <regex>
-#include <string>
```

Comment:
std::string is being used in this file; It is better to include what you use (instead of relying on other files to include what you need).